layers/meta-balena-allwinner/recipes-kernel/linux/linux_4.14%.bbappend

```diff
@@ -127,3 +128,7 @@ RESIN_CONFIGS_append_orange-pi-lite = " \
 RESIN_CONFIGS[8189fs] ?= " \
     CONFIG_RTL8189FS=m \
 "
+
```

Comment:
What is the build error?

Reply:
ERROR: linux-4.14.14+gitAUTOINC+9c0bf98471-r0 do_package: QA Issue: linux: Files/directories were installed but not shipped in any package:
  /boot/sun8i-h3-fixup.scr
Please set FILES such that these items are packaged. Alternatively if they are unneeded, avoid installing them or delete them within do_install

Reply:
How come this new file got generated now? We did not change the kernel in this PR, did we?

Reply:
Since we are not using this kernel version anymore, I guess we can drop this commit?

Reply:
If we would like to go back to kernel 4.14 in the future, for some reason, we will need this commit because the error will still be there if I drop this.

Reply:
Yes we will need this. But let's add it when we need it, I want to avoid stashing stuff we don't use.

Reply:
Ok. Will drop this commit.